node/pkg/watchers/sui/watcher.go

```diff
@@ -275,24 +319,55 @@ func (e *Watcher) inspectBody(logger *zap.Logger, body SuiResult, isReobservatio
 		IsReobservation:  isReobservation,
 	}
 
+	err = e.verifyAndPublish(ctx, observation, *body.ID.TxDigest, logger)
+
+	if err != nil {
+		logger.Error("Message publication error",
```

Comment:
There should be a Prometheus metric for this so that guardians can easily track it.